lib/controllers/hosted-skill-controller/clone-flow.js

```diff
@@ -74,7 +76,12 @@ function _gitCloneWorkflow(projectPath, repositoryUrl, verbosityOptions, profile
     const gitClient = new GitClient(projectPath, verbosityOptions);
     gitClient.init();
     const credentialHelperPath = `ask util git-credentials-helper --profile ${profile}`;
```

Comment:
why do we still need this credential helper as well?

Reply:
We can remove it and just execute the script, I don't have a strong opinion on this.